app/routes/__public/oauth/$id.tsx

```diff
@@ -221,11 +251,12 @@ const getAnonymousAccessToken = async ({
         uuid: sql`UUID()`,
         value: response.body.accessToken,
         code,
-        installationId: searchParams.installation_id,
-        userId: searchParams.installation_id,
+        installationId: installation_id || "",
+        userId: installation_id || "",
+        state: state || "",
```

Comment:
damn. this is probably fine for now but I sense that this may need some rethinking down the road - at the very least I want to build out some tests securing this flow